### PR TITLE
Add support for GeoIP2 anonymous database

### PR DESCRIPTION
**Pull request by mdevreugd**
Thanks for contributing to Logstash! If you haven't already signed our CLA, here's a handy link: https://www.elastic.co/contributor-agreement/

### CODE REVIEW

File: src/main/java/org/logstash/filters/GeoIPFilter.java [hunk 1]
```diff
@@ -401,4 +408,48 @@ private boolean applyGeoData(Map<String, Object> geoData, Event event) {
...
+    Map<String, Object> geoData = new HashMap<>();
+    for (Fields desiredField : this.desiredFields) {
+      switch (desiredField) {
+        case IP:
+          geoData.put(Fields.IP.fieldName(), ipAddress.getHostAddress());
+          break;
+        case ANONYMOUS_IS_ANONYMOUS:
+          Boolean is_anonoymous = response.isAnonymous();
+          if (is_anonoymous != null) {
+            geoData.put(Fields.ANONYMOUS_IS_ANONYMOUS.fieldName(), is_anonoymous);
```
[the remark this answers is not in the document]
Comment (zeysh) [reply src/main/java/org/logstash/filters/GeoIPFilter.java]: :+1:

[the remark this answers is not in the document]
Comment (zeysh) [reply src/main/java/org/logstash/filters/GeoIPFilter.java]: :+1:

**Comment by 9uill4ume** [comment]
@mdevreugd : how can we do so that our changes are validated and merged ? I can spend time on it, if you want.

**Comment by psdjc** [comment]
+1

File: vendor.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [
     {
         "url": "http://geolite.maxmind.com/download/geoip/database/GeoLite2-City.mmdb.gz",
```
Comment (Jokawild88) [inline vendor.json:3]: Que

**Review by Jokawild88 — CHANGES_REQUESTED** [review]

**Comment by edmocosta** [comment]
Closing this PR as this feature was added through https://github.com/logstash-plugins/logstash-filter-geoip/pull/223.

**Pull request closed without merging**